app/views/pages/_outside_links.html.haml

```diff
@@ -0,0 +1,21 @@
+%nav.fa-container
+  %a{:href => "https://github.com/444c43"}
+    %span web
+    %span.icon.fa-github
+    %span developer
+  &nbsp;&bull;&nbsp;
+  %a{:href => "http://twitter.com/2toneDL"}
+    %span social
+    %span.icon.fa-twitter
+    %span bunny
+  %p
+  %a{:href => "http://main.nationalmssociety.org/goto/2toneDL"}
+    %span avid
+    %span.icon.fa-bicycle
+    %span cyclist
+  &nbsp;&bull;&nbsp;
+  %a{:href => "#"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace